test/setupFile.js

```diff
@@ -3,6 +3,7 @@ import 'babel-polyfill';
 jest.mock('~/components/shell/shell');
 
 global.fetch = require('jest-fetch-mock');
+global.regeneratorRuntime = regeneratorRuntime;
```

Comment:
OIC, cool!

And it looks like the line above could be switched to `require('fetch-vcr')` and you can record the HTTP responses from Django 😄